kbase-extension/static/kbase/js/kbwidget.js

```diff
@@ -27,7 +27,7 @@ return KBWidget(
         version : "1.0.0",  //future proofing, but completely unused
 
         _accessors : [  //optional. A list of values to automatically create setter/getters for.
-            'foo'       //you'll now be able to store something at $widget.foo('newValue') and access it via var foo = $widget.foo();
```

Comment:
Generally I have tried to add comments whenever there's a change that isn't var -> let/const, removing an unused argument, or something else that's similarly trivial.